src/Common/midi/RtMidiDriver.cpp

```diff
@@ -115,6 +174,45 @@ QString RtMidiDriver::getInputDeviceName(uint index) const{
     return "";
 }
 
+QString RtMidiDriver::getOutputDeviceName(uint index) const{
+    RtMidiOut rtMidi;
+    if(index < rtMidi.getPortCount())
```

Comment:
Hi, just a small code format tip: Add a space between if/for and the opening parenthesis.